operator/notifications/notifier.go

```diff
@@ -0,0 +1,8 @@
+// Notifier
+
+package notifications
+
+// Notifier sends a notification
+type Notifier interface {
```

Comment:
Interfaces should be defined where they are used, rather than where they are implemented. 

https://github.com/Wattpad/highlander/wiki/Go-best-practices:--avoiding-%22Java-style%22-interfaces